src/main/java/com/example/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -40,6 +42,7 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
 
     @ExceptionHandler(Exception.class)
     protected ResponseEntity<Object> handleException(Exception ex) {
+        log.error("error message", ex);
```

Comment:
I think the logic was that we log unexpected exceptions for which we return 500 status because it's a programmer's error, but don't log client exceptions because they are common and it will occupy a lot of memory. So makes sense for me.